### PR TITLE
[PATCH v2] example: debug: add debug print calls

### PR DESCRIPTION
**Pull request by psavol**
Add calls to odp_pktio_print(), odp_packet_print_data(), odp_ipsec_print() and odp_sys_config_print().

### CODE REVIEW

File: example/debug/odp_debug.c [hunk 1]
```diff
@@ -285,49 +345,67 @@ int main(int argc, char *argv[])
...
+		global->ipsec   = 1;
 	} else {
 		if (parse_options(argc, argv, global))
 			return -1;
 	}
 
 	if (odp_init_global(&inst, NULL, NULL)) {
-		printf("Global init failed.\n");
+		ODPH_ERR("Global init failed.\n");
 		return -1;
```
Comment (matiaselo) [inline example/debug/odp_debug.c:361]: While your are cleaning the error messages, you could also use standard return codes. E.g. `exit(EXIT_FAILURE)`.

File: example/debug/odp_debug.c [hunk 2]
```diff
@@ -260,21 +271,70 @@ static int queue_debug(void)
...
+	}
+
+	return 0;
+}
+
+static int ipsec_debug(void)
+{
+	printf("\n");
+	odp_ipsec_print();
+
```
Comment (matiaselo) [inline example/debug/odp_debug.c:334]: When have also`odp_ipsec_sa_print()`, but support for this can be added in a separate commit.

**Review by matiaselo — APPROVED** [review]

File: example/debug/odp_debug.c (hunk 2 above)
Comment (psavol) [reply example/debug/odp_debug.c:334]: Yes, left that for the next round. It needs considerable number of lines (to check capabilities) to successfully create a SA of any supported type.

File: example/debug/odp_debug.c (hunk 1 above)
Comment (psavol) [reply example/debug/odp_debug.c:361]: OK. I'll do that as well.

**Pull request merged**